os-autoinst-obs-auto-submit

```diff
@@ -102,6 +102,12 @@ make_git_submit_request() {
     #       For `git push` to work (done below) you need to deploy an SSH key on the Gitea instance.
     #       Checkout https://opensuse-commander.readthedocs.io/en/latest/git-obs-quickstart.html for details.
     forked_repo=$($git_obs repo fork "pool/$package" 2>&1 > /dev/null | perl -ne 'print for /Fork .*: (.*)/g')
+    if [[ -z $forked_repo ]]; then
```

Comment:
It would probably be good to print the actual output of the fork command, which should already explain the problem.
So catch the output and only after that apply your regex to get the repo_name
Otherwise we are blindly guessing about the cause of the error